src/web/EmployerFavouritesWeb/Startup.ConfigureServices.cs

```diff
@@ -45,6 +45,9 @@ public void ConfigureServices(IServiceCollection services)
 
             services.AddMediatR(typeof(Startup).Assembly);
 
+            services.AddHealthChecks()
```

Comment:
Shouldnt the FatApi healthcheck be added in here too?

Reply:
👍